### ISSUE TITLE
Thinking about plots for the manuscript 

### ISSUE THREAD

**Issue by venkatachalapathy** [WheezePuppet/specnet#27]
So, what are the plots that we might end of including in the manuscript? It depends on the scientific questions we are going to ask but thinking in terms of plots might help us ask more focused questions. 

One 3D plot I had in mind was histogram of size distribution of proto-institutions versus network characteristics 
+ lambda parameter for ER
+ alpha for SF
+ scaling parameter for SW 

Another one is the wealth distribution at the end point of our simulation versus the network characteristics

Anything else?

**Comment by divilian** [comment]
I'm a big fan of "Gini coefficient over time" for a single run of the simulation, with many overlapping line plots on different runs with the same parameters to show general trends.

When you say "size distribution of proto-institutions," do you mean a histogram that shows how many protos have exactly 2 members, how many have 3, how many have 4, etc.? Yes, that could be cool.

In general, I shy away from 3d plots because I feel that most humans (including myself) have difficulty meaningfully interpreting them. I feel their main value-add is to look cool.

For "wealth distribution at the end point of our sim" do you mean the entire Lorentz curve? Or just a Gini?

Pinging @will-nehrboss on this convo as well since we'll both probably be coding it.

**Comment by wkneh** [comment]
Once you nail down what this will be, send me an email to let me know what
aspects I can build/code.
Look forward to it.

Will

**Comment by venkatachalapathy** [comment]
Yes, I was thinking more in terms of a heat map where the z-axes is dependent on (x,y) and colored.  And also some kind of histogram comparision test to find out if the any two interesting histograms are statistically different from one another.

**Comment by divilian** [comment]
So now that we have an animated wealth-distribution-over-time histogram, I'm realizing that doesn't exactly work for the manuscript. :) So I guess I should build a 2d one after all. Create new issue #28 for this.
